src/editors/data/redux/problem/reducers.js

```diff
@@ -12,8 +12,10 @@ const initialState = {
   answers: [],
   correctAnswerCount: 0,
   groupFeedbackList: [],
+  generalFeedback: '',
   additionalAttributes: {},
   settings: {
+    randomization: null,
```

Comment:
Is this value tied to the randomization field in "Advanced Settings"? If yes, do we want to include the hyperlink to the "AdvancedSettings" page and have the default value match what is in the settings?

Reply:
Oh, yeah that is set at the course level, and we probably should do that. nice catch.

Reply:
> do we want to include the hyperlink to the "AdvancedSettings" page 

 -- this doesn't match mocks so I'm gonna skip it.